src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -514,8 +514,7 @@ public VisualTransformPolicyType CornerRadiusPolicy
         /// </code>
         /// </para>
         /// </remarks>
-        /// This will be public opened after ACR done. Before ACR, need to be hidden as inhouse API.
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
```

Comment:
I would rather put "Note that..." in the \<remarks\> section and write "*the* Border property". But that's not a big deal.

Reply:
Fix done.